_config.yml

```diff
@@ -13,12 +13,16 @@ exclude_from_localizations: ['javascript', 'images', 'css']
 
 # Contact form production values (overridden in Federalist for preview branches)
 contact_form_captcha_enabled: true
-contact_form_action: "https://webto.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8"
-contact_form_orgid: "00DU0000000Leux"
+contact_form_action: 'https://webto.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8'
+contact_form_orgid: '00DU0000000Leux'
+
+# Contact page phone number enabled
+contact_phone_number_enabled: false
+contact_phone_number: ''
```

Comment:
Is the plan for this configuration to be temporary? If yes, what's the timeline for removing it? If no, I'd worry that setting the default to `false` could become misleading for developers due to a mismatch of content between the local and live versions of the site.